ePBFT/src/pbft/libbyz/LedgerReplay.cpp

```diff
@@ -36,54 +27,70 @@ std::unique_ptr<Pre_prepare> LedgerReplay::process_data(
   PBFT_ASSERT(
```

Comment:
Is this an issue? I reckon the function should not do anything if data is empty?

Reply:
This was left here from before this evolved, I will remove it